.circleci/config.yml

```diff
@@ -301,6 +301,16 @@ commands:
             circleci tests glob "tests/**/*.spec.ts" | circleci tests split --split-by=timings > /tmp/tests-to-run
             yarn test:ci $(cat /tmp/tests-to-run)
           no_output_timeout: 15m
+      - run:
+          name: Create circleci_parameters.json
+          working_directory: ~/workbench/e2e
+          command: |
+            echo "{ \"circleBranch\": \"${CIRCLE_BRANCH}\", \"circleJob\": \"${CIRCLE_JOB}\", \"circleBuild\": \"${CIRCLE_BUILD_NUM}\", \"circleBuildUrl\": \"${CIRCLE_BUILD_URL}\", \"circleNodeIndex\": \"${CIRCLE_NODE_INDEX}\" }" \
+              > ./logs/circleci_parameters.json
+      - run:
+          name: Combine circleci_parameters.json and test-results.json
```

Comment:
nit: I would not re-encode the file names here in the description, e.g. "Add CircleCI job data to Junit test results".

Reply:
Done